Library/Formula/v8.rb

```diff
@@ -21,6 +21,7 @@ class V8 < Formula
 
   depends_on :python => :build # gyp doesn't run under 2.6 or lower
   depends_on "readline" => :optional
+  depends_on "icu4c" => :recommended
```

Comment:
My reasoning is: it's not default now and I'd rather not add a dependency on people without much demand.